.github/workflows/mainnet.yml

```diff
@@ -0,0 +1,38 @@
+name: Mdw Frontend Mainnet
+
+on:
+  push:
+    branches: 
+      - master
+    tags:
+      - 'v*'
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+  
+    name: Build Docker
+    steps:
+
+      - uses: actions/checkout@v2
+  
+      - name: Log in to Docker Hub
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASS }}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: nginx.Dockerfile
+          build-args: |
+            NUXT_APP_NODE_URL: "https://mainnet.aeternity.io/mdw"
```

Comment:
newer versions have some additional ENV vars
```
    nodeURL: process.env.NUXT_APP_NODE_URL || 'https://mainnet.aeternity.io/v3',
    middlewareURL: process.env.NUXT_APP_MDW_URL || 'https://mainnet.aeternity.io/mdw',
    middlewareWS: process.env.NUXT_APP_NODE_WS || 'wss://mainnet.aeternity.io/mdw/websocket',
    networkName: process.env.NUXT_APP_NETWORK_NAME || 'MAINNET',
    otherDeployments: process.env.NUXT_APP_OTHER_DEPLOYMENTS || 'TESTNET@https://explorer.testnet.aeternity.io',
    enableFaucet: process.env.NUXT_APP_ENABLE_FAUCET || false,
    faucetAPI: process.env.NUXT_APP_FAUCET_API || 'https://testnet.faucet.aepps.com/account',
    APIDocs: process.env.NUXT_APP_API_DOCS || 'https://github.com/aeternity/ae_mdw#http-endpoints'

Reply:
Okey , thanks , we can sync on that.